corehq/apps/es/users.py

```diff
@@ -163,20 +163,16 @@ def primary_location(location_id):
     )
 
 
-def location(location_id, include_web_users=False):
+def location(location_id):
```

Comment:
Nice, thanks!  Did it look like it was okay to change the behavior of the other usages of this filter?